### PR TITLE
Ashpak/feature v3 swagger imagesnapshot ewc 197

### PR DESCRIPTION
**Pull request by alexandershpak**

### CODE REVIEW

File: specs/swagger.yml [hunk 1]
```diff
@@ -1236,6 +1236,75 @@ paths:
           description: Not Found
         '500':
           description: Internal Server Error
+  '/iofog/microservices/{id}/imageSnapshot':
```
Comment (baghbidi) [inline specs/swagger.yml]: `/iofog/microservices/{id}/image-snapshot`

File: specs/swagger.yml [hunk 2]
```diff
@@ -1236,6 +1236,75 @@ paths:
...
+          description: User Token
+          required: true
+          type: string
+        - in: path
+          name: id
+          description: Microservice Id
+          required: true
+          type: string
+      responses:
+        '201':
```
Comment (baghbidi) [inline specs/swagger.yml]: `200`

File: specs/swagger.yml [hunk 3]
```diff
@@ -1236,6 +1236,75 @@ paths:
...
+          description: Created
+          schema:
```
Comment (baghbidi) [inline specs/swagger.yml]: Why should it return an object?
Response type must be `file` followed by appropriate `produce` property.

File: specs/swagger.yml (hunk 1 above)
Comment (alexandershpak) [reply specs/swagger.yml]: fix

File: specs/swagger.yml (hunk 2 above)
Comment (alexandershpak) [reply specs/swagger.yml]: fix

**Pull request merged**